Formula/gazebo7.rb

```diff
@@ -34,12 +34,6 @@ class Gazebo7 < Formula
   depends_on "gdal" => :optional
```

Comment:
We should deprecate / disable / remove `gazebo7`

Reply:
I will deprecate it soon